src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -267,7 +269,7 @@ internal void PublishWorkerProcessReadyEvent(FunctionEnvironmentReloadResponse r
             _eventManager.Publish(wpEvent);
         }
 
-        internal void PublishRpcChannelReadyEvent(RpcEvent initEvent)
+        internal void SendDownloadDependencyRequest(RpcEvent initEvent)
```

Comment:
Add more info level logs. Info is default level for system logs. We should be able to figure out from the logs,  when request to download dependencies was sent to the worker and if it succeeded.